### DISCUSSION TITLE
Trouble with VDataTable: TypeError: item.props is undefined

### DISCUSSION THREAD

**Discussion by capoaira** [vuetifyjs/vuetify#19067] [Q&A]
I have a project which uses the `v-data-table` from labs (version 3.1.10). Now I need functions of the table which are not included in the labs, but in the last release (version 3.4.10). So I updated vuetify in the project, but now I'm getting the Error `TypeError: item.props is undefined` from other usages of the table.
Any Ideas how to fix that?

```jsx
<template #item.created="{ item }">
  <p>
    {{ item.props.title.created?.split("T")[0] }}
  </p>
</template>
```

Maybe this question is related to #18784

**Comment by KaelWD** [comment]
https://github.com/vuetifyjs/vuetify/commit/283a18f31f4da13877e59c51282a61a93bb6aaa2
